### PR TITLE
#10. Make Column Config popup options configuration driven.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**

### CODE REVIEW

File: src/components/AdminPanel.jsx [hunk 1]
```diff
@@ -86,7 +80,10 @@ class AdminPanel extends Component {
     this.props.clearSearchResultsAction();
     const { selectSearchOption, searchText } = this.state.search;
     if (!isEmpty(selectSearchOption) && !isEmpty(searchText)) {
-      this.props.fetchSearchResultsAction(adminPassword, selectSearchOption, searchText);
+     const adminKey = adminPassword;
+     const searchKey = selectSearchOption;
+     const searchValue = searchText;
+      this.props.fetchSearchResultsAction({ adminKey, searchKey, searchValue });
```
Comment (mohitsaladiya) [inline src/components/AdminPanel.jsx]:
```suggestion
      this.props.fetchSearchResultsAction({ adminKey: adminPassword, searchKey: sekectSearchOption, searchValue: searchText });
```

File: src/components/AdminPanel.jsx [hunk 2]
```diff
@@ -86,7 +80,10 @@ class AdminPanel extends Component {
     this.props.clearSearchResultsAction();
     const { selectSearchOption, searchText } = this.state.search;
     if (!isEmpty(selectSearchOption) && !isEmpty(searchText)) {
-      this.props.fetchSearchResultsAction(adminPassword, selectSearchOption, searchText);
+     const adminKey = adminPassword;
```
Comment (mohitsaladiya) [inline src/components/AdminPanel.jsx]:
```No need of this. Please have a look at below suggestion.
     const adminKey = adminPassword;
     const searchKey = selectSearchOption

```

File: src/components/ColumnConfig.jsx [hunk 3]
```diff
@@ -159,7 +171,7 @@ class ColumnConfig extends Component {
                   <span className="select-none-wrapper">Select All</span>
                 </label>
               </div>
-              {this.renderColumnOptions()}
+              {this.renderColumnOptions()};
```
Comment (mohitsaladiya) [inline src/components/ColumnConfig.jsx]: is this semicolon needed?

File: src/components/AdminPanel.jsx (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/AdminPanel.jsx]: Done.

File: src/components/ColumnConfig.jsx (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/ColumnConfig.jsx]: No, I have removed it.

**Pull request merged**